src/java.base/share/classes/java/io/Console.java

```diff
@@ -148,100 +147,6 @@ public Reader reader() {
         throw newUnsupportedOperationException();
     }
 
-    /**
```

Comment:
The copyright header will need to be updated on the changes to java.base as they seem to be first update to these sources in 2025.